app/components/Navigation.jsx

```diff
@@ -8,7 +10,28 @@ export default class Navigation extends Component {
   static propTypes = {
     currentLocation: React.PropTypes.object
   }
-  
+
+  constructor () {
+    super();
+    this.state = {
+      loggedIn: null,
+    };
+    this.updateAuth = this.updateAuth.bind(this);
```

Comment:
Can you rename to `_updateAuth()` instead?

One convention I'm trying to keep is to prefix all internal APIs with underscore to avoid confusion ([as a reference](https://github.com/venicegeo/bf-ui/blob/master/app/components/JobStatusList.jsx#L17)).